README.md

```diff
@@ -100,6 +100,47 @@ var client = ads.connect(options, function() {
 })
 ```
 
+
+### MultiRead something
+
+```javascript
+        this.client.multiRead(
```

Comment:
I think the example should be like any other

var client = ads.connect(options, function() {
   client.multiRead(
....
}

Reply:
the "this.end ()" is also missing

Reply:
excuse me sometimes meticulous